src/publishMain.ml

```diff
@@ -519,6 +533,11 @@ module Args = struct
     value & flag &
     info ["no-confirmation"] ~docs ~doc:
       "Skip all confirmations. This should only ever be used in very special circumstances, like automated publishing CI workflows where measures to ensure the submission has been reviewed have already taken place."
+
+  let exclude =
+    value & opt (some (list string)) None &
```

Comment:
As a mild type simplification, why not just have `string list` where the default is `[]` (there doesn’t seem any value in propagating `None` and `Some []`)

Reply:
done